ui/src/app/pages/workspace/research-purpose.tsx

```diff
@@ -71,6 +92,21 @@ export const ResearchPurpose = withCurrentWorkspace()(
                               style={styles.editIcon}/>
         </Clickable>
       </div>
+      {isOwner && !workspace.researchPurpose.researchPurposeReviewed && <FlexRow style={styles.reviewPurposeReminder}>
+        <ClrIcon style={{color: colors.warning, marginLeft: '0.3rem'}} className='is-solid'
+        shape='exclamation-triangle' size='25'/>
+        <FlexColumn style={{paddingRight: '0.5rem', paddingLeft: '0.5rem', color: colors.primary}}>
+        <label style={{fontWeight: 600, fontSize: '14px', flex: 1}}>
+        Please update or verify the following Research Purpose.</label>
+        </FlexColumn>
+        <div style={{marginLeft: 'auto', marginRight: '0.5rem'}}>
+        <a style={{marginRight: '0.5rem'}} onClick={() => updateWorkspace(workspace)}>Looks
```

Comment:
I don't have access to the design mock - does it really say 'looks good'?

Reply:
Yes